### PR TITLE
[RTD-459] Add RTD transaction model

### PR DESCRIPTION
**Pull request by TommasoLencioni**
#### Description

This PR proposes to add the model of a transaction ingested by RTD.
#### List of Changes
- add RTD transaction model

#### Motivation and Context
Decrypted files need to be validated. To do so for RTD transactions their DTO must be available.

#### How Has This Been Tested?

- [ ] Unit Test Suite
- [ ] Integration Test Suite
- [ ] Api Tests
- [ ] Load Tests

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/model/RtdTransaction.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 14)
+  @Pattern(regexp = "[a-zA-Z0-9]{1,5}", message = "MCC length must be 5 alphanumeric char at max")
+  String mcc;
+
+  @CsvBindByPosition(position = 15)
+  @Pattern(regexp = "[a-zA-Z0-9]{11}|[a-zA-Z0-9]{16}",
+      message = "Fiscal code must be 11 or 16 alphanumeric char")
+  String fiscalCode;
```
Comment (and-mora) [inline src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/model/RtdTransaction.java:134]: is fiscalCode not mandatory?

File: src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/model/RtdTransaction.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  String vat;
+
+  @NotNull
+  @NotBlank
+  @CsvBindByPosition(position = 17)
+  @Pattern(regexp = "\\d{2}", message = "Pos type must match [0-9]{2}")
+  String posType;
+
+  @CsvBindByPosition(position = 18)
+  String par;
```
Comment (GiovanniMancini) [inline src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/model/RtdTransaction.java:148]: @TommasoLencioni Maybe the PAR could be validated following the structure reported here
https://www.securetechalliance.org/secure/events/20160404/CORAL-SEA-1-2_WED_1045_SRIVASTAVA_SCA-Payment-Summit-Tokenization-PAR-presentation-April-6-Chandra-Srivastava-Visa.pdf

**Review by GiovanniMancini — APPROVED** [review]

File: src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/model/RtdTransaction.java (hunk 2 above)
Comment (TommasoLencioni) [reply src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/model/RtdTransaction.java:152]: Done, thank you!

File: src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/model/RtdTransaction.java (hunk 1 above)
Comment (TommasoLencioni) [reply src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/model/RtdTransaction.java:136]: I enhanced fiscal code validation, thank you!

**Pull request merged**